README.md

```diff
@@ -6,4 +6,39 @@ A web scraper tool to get data for evaluating Wellcome impact.
 
 ## Usage
 
-See the wiki: https://github.com/wellcometrust/wsf-web-scraper/wiki
+To deploy this scraper yourself, see the wiki: https://github.com/wellcometrust/wsf-web-scraper/wiki
+
+This scraper can also be deployed more easily using Docker.
+
+## Output Formating
+
+The outputed file is meant to contains a number a different fields, which can vary depending on the scraped provider.
+
+It will always have the following attribute, thought:
+
+|Attribute|Description|
+|---------|-----------|
+|title    | a string containing the document title|
+|uri      | the url of the document|
+|pdf      | the name of the file|
+|sections | a json object of section names, containing the text extracted from matching sections|
+|keywords | a json object of keywords, containing the text extracted from matching text|
+|hash     | a md5 digest of the file|
+|provider | the provider from where the file has been downloaded|
+
+Some providers will have additional parameters:
+
+### WHO
+
+|Attribute|Description|
+|---------|-----------|
+|year     | the publication year of the document|
+|types    | an array containing the who type associated with the document|
+|subjects | an array containing the who subjects of the document|
+|authors  | an array containing the authors (from WHO)|
+
+### Nice
+
+|Attribute|Description|
+|---------|-----------|
+|year     | the publication year of the document|
```

Comment:
do you anticipate that they might not all have years then?

Reply:
It's a sure thing, I didn't manage to get the years for uncief, gov and msf

Reply:
ah that's annoying! Good to know.

Reply:
If it's very important for you to have it, I probably manage somehow, but it's not easy on the three new websites